test/integration/basic_nsmd_deploy_test.go

```diff
@@ -47,20 +47,20 @@ func testNSMDDdataplaneDeploy(t *testing.T, nsmdPodFactory func(string, *v1.Node
 		return
 	}
 
-	var pods []*v1.Pod
+	var podsVal []*v1.Pod
 
 	for i, node := range nodes {
 		nsmdPodName := fmt.Sprintf("nsmd-%d", i+1)
-		dataPlanePodName := fmt.Sprintf("nsmd-dataplane-%d", i+1)
+		dataPlanePodName := fmt.Sprintf("nsmd-dataplane%d", i+1)
```

Comment:
No. Please let's keep unified naming, We had #817 treating this across all tests.

Overall the changes in this file shoudl nto be part of this PR.